java/com/google/devtools/moe/client/dvcs/git/GitRevisionHistory.java

```diff
@@ -89,7 +89,7 @@ public RevisionMetadata getMetadata(Revision revision) {
     }
 
     // Format: hash, author, fullAuthor, date, parents, full commit message (subject and body)
-    String format = Joiner.on(LOG_DELIMITER).join("%H", "%an", "%aN <%cE>", "%ad", "%P", "%B");
+    String format = Joiner.on(LOG_DELIMITER).join("%H", "%an", "%aN <%aE>", "%ad", "%P", "%B");
```

Comment:
The `git log` execution is what determines the commit information.  I've ran this locally, and it solves the problem of commits being pushed by svcscm for an author getting the e-mail wrong.